### DISCUSSION TITLE
https://cucoders.dev/publicaciones/manuelernestog/gu-a-para-prompting-c-mo-obtener-mejores-resultados-de-los-modelos-de-lenguaje/

### DISCUSSION THREAD

**Discussion by giscus** [CuCodersCommunity/cucoderscommunity.github.io#74] [Page Comments]
# https://cucoders.dev/publicaciones/manuelernestog/gu-a-para-prompting-c-mo-obtener-mejores-resultados-de-los-modelos-de-lenguaje/

En este artículo, discutiremos algunas guías para el prompting efectivo, incluyendo cómo escribir instrucciones claras y específicas, cómo darle al modelo tiempo para pensar, y cómo abordar algunas de las limitaciones comunes de los modelos.

https://cucoders.dev/publicaciones/manuelernestog/gu-a-para-prompting-c-mo-obtener-mejores-resultados-de-los-modelos-de-lenguaje/

**Comment by josernesto1989** [comment]
Para que de respuestas más acertadas puedes decirle al inicio de una conversación "Asume que eres un desarrollador seinor en Python,  Django y apirest", luego de esto las preguntas que le hagas las responderá con el perfil que le diste sin necesidad de especificarlo, por ejemplo le pides un factorial y lo hará en Python directamente. Sirve para otros campos que no sea la programación. Es una buena practica que los primeros proms sean para crear el perfil de la conversación.
